src/app/form/form.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.css']
+})
+export class FormComponent {
+  name:String='';
```

Comment:
format the line as following `name: String = '';`

Reply:
wow , thanks alot.